app/build.gradle.kts

```diff
@@ -53,6 +60,13 @@ android {
 
         versionCode = getVersionCode()
         versionName = getVersionName()
+        resourceConfigurations.addAll(
+            listOf(
+                "en",
+                "pl",
+                "pt"
+            )
```

Comment:
What does this setting do?

Reply:
It does both things:

* Removes unused resources: <https://developer.android.com/build/shrink-code#unused-alt-resources>
* defines app languages: <https://developer.android.com/guide/topics/resources/app-languages#gradle-config>

Reply:
Very good!